src/js/binaryen.js-post.js

```diff
@@ -4768,6 +4796,146 @@ Module['I31Get'] = makeExpressionWrapper({
   }
 });
 
+// ElementSegment wrapper
+Module['ElementSegment'] = (() => {
+  // Closure compiler doesn't allow multiple `Function`s at top-level, so:
+  function Function(func) {
```

Comment:
Should all these `Function` be `ElementSegment`?

Reply:
Probably. I wasn't sure if these constructors should match the key or if Function was meant to be generic (due to the comment). I'll change them.